src/update.c

```diff
@@ -247,7 +261,7 @@ update_job_new (gpointer owner,
 
 	job = g_new0 (struct updateJob, 1);
 	job->owner = owner;
-	job->request = request;
+	job->request = UPDATE_REQUEST (g_object_ref (request));
```

Comment:
Previously, it looked like the `updateJob` took ownership of the request. Here the job takes a reference on the request, but there is already a reference and the caller doen't unref it, so it looks like the UpdateRequest will be leaked, as the reference count will still be one after `update_job_free`.

Reply:
Also fixed. Thanks for catching this!